go/worker/event/server.go

```diff
@@ -199,20 +198,11 @@ func shutdown(pidPath string, server cleanable) {
 	slog.Info(fmt.Sprintf("save stats to %s", statsPath))
 	if s, err := json.MarshalIndent(snapshot, "", "\t"); err != nil {
 		slog.Error(fmt.Sprintf("error: %s", err))
-		rc = 1
 	} else if err := ioutil.WriteFile(statsPath, s, 0644); err != nil {
 		slog.Error(fmt.Sprintf("error: %s", err))
-		rc = 1
 	}
 
-	slog.Info(fmt.Sprintf("Remove %s.", pidPath))
-	if err := os.Remove(pidPath); err != nil {
-		slog.Error(fmt.Sprintf("error: %s", err))
-		rc = 1
-	}
-
-	slog.Info(fmt.Sprintf("Exiting worker (PID %d)", os.Getpid()))
-	os.Exit(rc)
+	slog.Info(fmt.Sprintf("Printed final stats of worker (PID %d)", os.Getpid()))
```

Comment:
A lot of current code uses this format (slog.Info(fmt.Sprintf(...)), but that's just because I was porting from log.Printf.

A better pattern is like this:
slog.Info("Printed final stats of worker", "pid", os.Getpid())